src/compounds/legacy-product-table/src/index.tsx

```diff
@@ -603,6 +604,17 @@ const TelephoneInfo: React.FC<TelephoneInfoProps> = ({ telephone }) => {
   )
 }
 
+// Shared type with phone-number-modal
+interface PhoneNumberModalInfo {
+  phoneNumber: string
+  logoUrl: string
+  logoDescription: string
+  termsAndConditions: string
+  openingTimes?: string[]
+  url?: string
+  complianceText: string[]
+}
+
```

Comment:
You should be able to export the type from the phone number component: https://github.com/uswitch/trustyle/blob/bcd2a2afb248eea22fb31e2fd1b8232b7e53e87b/src/compounds/phone-number-modal/src/index.tsx#L7

instead of repeating the type twice